packages/office-ui-fabric-react/src/components/Separator/examples/Separator.Basic.Example.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Separator } from 'office-ui-fabric-react/lib/Separator';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+
+export class SeparatorBasicExample extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    const message = 'Today';
+
+    const verticalStyle = mergeStyles({
+      height: '200px'
+    });
+
+    return (
+      <Stack gap={15}>
+        <Stack gap={10}>
+          <Text>Horizontal center aligned</Text>
+          <Separator>Today</Separator>
```

Comment:
`{message}`?

Reply:
I'll call it `content` instead while I'm at it too!